__tests__/app.test.js

```diff
@@ -27,3 +27,18 @@ describe('/api/topics', () => {
     return request(app).get('/api/nonsense').expect(404);
   });
 });
+describe('/api', () => {
+  test('get object describing all existing endponts', () => {
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body).toBe('object');
+        for (const key in body) {
```

Comment:
I would look at the notes for this, we can just read the JSON file and make sure the response === that, that way it will always work and be dynamic. We care more that the documentation is being sent rather than what is on it